srtcore/group.cpp

```diff
@@ -448,10 +448,12 @@ template <class ValueType>
 static void importOption(vector<CUDTGroup::ConfigItem>& storage, SRT_SOCKOPT optname, const ValueType& field)
 {
     ValueType default_opt      = ValueType();
-    int       default_opt_size = sizeof(ValueType);
+    const int default_opt_size = sizeof(ValueType);
     ValueType opt              = field;
-    if (!getOptDefault(optname, (&default_opt), (default_opt_size)) || default_opt != opt)
+    int opt_size               = default_opt_size;
+    if (!getOptDefault(optname, (&default_opt), (opt_size)) || default_opt != opt)
     {
+        SRT_ASSERT(default_opt_size == default_opt_size);
```

Comment:
What exactly the check for equality of a variable to itself has to improve here?

Reply:
Sorry, that was a typo.